oak-blob-cloud-azure/src/test/java/org/apache/jackrabbit/oak/blob/cloud/azure/blobstorage/AzureBlobStoreBackendTest.java

```diff
@@ -18,8 +18,10 @@
  */
 package org.apache.jackrabbit.oak.blob.cloud.azure.blobstorage;
 
-import com.microsoft.azure.storage.StorageException;
```

Comment:
These tests should be moved to the v8 package as we still have the code living in the v8 package